layers/meta-balena-raspberrypi/conf/machine/include/revpi4.inc

```diff
@@ -9,9 +9,9 @@ SOC_FAMILY = "rpi:revpi"
 
 PREFERRED_PROVIDER_virtual/kernel = "linux-kunbus"
 
-KERNEL_MODULE_AUTOLOAD += "piControl"
+# KERNEL_MODULE_AUTOLOAD += "piControl"
```

Comment:
Why is this disabled?

Reply:
Probably because we had an out of tree kernel module, and the kernel we wanted to use was not yet integrated to be autoloaded.
I will enable it, rebuild the image, if everything went flawless I will update the PR.